src/client/datascience/data-viewing/types.ts

```diff
@@ -77,7 +78,7 @@ export interface IDataFrameInfo {
 
 export interface IDataViewerDataProvider {
     dispose(): void;
-    getDataFrameInfo(sliceExpression?: string): Promise<IDataFrameInfo>;
+    getDataFrameInfo(sliceExpression?: string, shouldUpdateCachedInfo?: boolean): Promise<IDataFrameInfo>;
```

Comment:
I think calling this refresh or something like that is better. 'shouldUpdateCachedInfo' means an external caller knows something of the internal implementation. How does an external caller know there's a cache? All the external caller should know is that the user hit the refresh button (or a refresh of the data is required)